Comp2023/src/main/java/frc/robot/commands/AutoPreloadAndEngageChargeStation.java

```diff
@@ -7,35 +7,48 @@
 import edu.wpi.first.wpilibj2.command.PrintCommand;
 import edu.wpi.first.wpilibj2.command.SequentialCommandGroup;
 import edu.wpi.first.wpilibj2.command.WaitUntilCommand;
+import frc.robot.Constants.GRConsts;
+import frc.robot.subsystems.Elbow;
+import frc.robot.subsystems.Gripper;
 import frc.robot.subsystems.Swerve;
+import frc.robot.subsystems.Wrist;
 
 /**
  *
  */
 public class AutoPreloadAndEngageChargeStation extends SequentialCommandGroup
 {
-  public AutoPreloadAndEngageChargeStation(Swerve swerve)
+  public AutoPreloadAndEngageChargeStation(Swerve swerve, Gripper gripper, Elbow elbow, Wrist wrist)
   {
-    setName("AutoPreloadAndEngageChargeStation");
+    setName("AutoPreloadAndLeaveCommunity");

```

Comment:
The name in setName should ALWAYS match the name of the command/class/filename. Not something different.